MMOCoreORB/src/server/zone/managers/city/CityManagerImplementation.cpp

```diff
@@ -755,14 +755,12 @@ void CityManagerImplementation::processCityUpdate(CityRegion* city) {
 
 		city->cleanupCitizens();
 
+		PlayerManager* playerManager = zoneServer->getPlayerManager();
 		ManagedReference<SceneObject*> mayor = zoneServer->getObject(city->getMayorID());
 
 		if (mayor != nullptr && mayor->isPlayerCreature()) {
-			Reference<PlayerObject*> ghost = mayor->getSlottedObject("ghost").castTo<PlayerObject*> ();
-
-			if (ghost != nullptr) {
-				ghost->addExperience("political", 750, true);
-			}
+			Reference<CreatureObject*> ghost = mayor->getSlottedObject("ghost").castTo<CreatureObject*> ();
+			playerManager->awardExperience(ghost, "political", 750, false);
```

Comment:
Removed a nullptr check here that should stay

Reply:
It is not necessary. The `ghost` pointer is not directly used like it was in the previous implementation. The PlayerManager's `awardExperience()` method will perform nullptr checks

Reply:
Resolving from discussion offline and in other comment